include/triton-shared/Conversion/TritonArithToLinalg/ConversionPatterns.hpp

```diff
@@ -949,6 +979,111 @@ struct PreciseDivConverter : public OpConversionPattern<triton::PreciseDivFOp> {
   }
 };
 
+struct CatConverter : public OpConversionPattern<triton::CatOp> {
```

Comment:
i think it would be useful to have a small lit test for each of these new converters so we can understand how each op is being lowered

Reply:
sure, will get back with a set of small tests for conversions

Reply:
added lit tests, but I have mixed feeling about them, it looks like we hardcoding implementation details instead of checking for the behavior

Reply:
You're right that they're hardcoding implementation details, but another benefit is they check 
that the ops are lowered as expected. The lit tests also make it a bit easier to reproduce any issues in the future since the triton IR is already in the mlir format; that means I can just debug issues using the standalone `triton-shared-opt --triton-arith-to-linalg` command instead of having to go through the triton compiler.